cmd/setup.go

```diff
@@ -9,25 +9,33 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var setUpCmd = &cobra.Command{
-	Use:   "setup",
-	Short: "Run OTel dependencies in Docker.",
-	RunE: func(cmd *cobra.Command, args []string) error {
-		if err := setUp(); err != nil {
-			return err
-		}
-		return nil
-	},
+var (
+	aspire   = false
+	setUpCmd = &cobra.Command{
+		Use:   "setup",
+		Short: "Run OTel dependencies in Docker.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			s := GetStackByFlags(aspire)
+			if err := setUp(s); err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+)
+
+func init() {
+	setUpCmd.PersistentFlags().BoolVarP(&aspire, "aspire", "", false, "Use .NET Aspire dashboard as OTel stack")
 }
 
-func setUp() error {
+func setUp(s Stack) error {
 	if err := checkDocker(); err != nil {
 		return err
 	}
-
-	composeFile := path.Join(otelConfigPath, "compose.yaml")
+	fn := s.GetComposeFileName()
```

Comment:
The name `fn` here makes me think you're holding a function in this variable.

Reply:
Renamed in 4dea516